kubernetes/application/blueprint.yaml

```diff
@@ -21,14 +21,19 @@ spec:
     prompt: "What is the version of the application that you want to deploy?"
     default: latest
 
-  - name: KubernetesResourcesFile
```

Comment:
If we are not letting the user choose the yaml files, then what is the point of letting them choose application name and version here?

Reply:
I've added it back so users can provide their own file again